src/VirtoCommerce.DemoSolutionFeaturesModule.Data/Models/DemoOrderLineItemEntity.cs

```diff
@@ -0,0 +1,48 @@
+using System.ComponentModel.DataAnnotations;
+using VirtoCommerce.DemoSolutionFeaturesModule.Core.Models;
+using VirtoCommerce.OrdersModule.Core.Model;
+using VirtoCommerce.OrdersModule.Data.Model;
+using VirtoCommerce.Platform.Core.Common;
+
+namespace VirtoCommerce.DemoSolutionFeaturesModule.Data.Models
+{
+    public class DemoOrderLineItemEntity: LineItemEntity
+    {
+        [StringLength(128)]
+        public string ConfiguredGroupId { get; set; }
+
+        public DemoOrderConfiguredGroupEntity ConfiguredGroup { get; set; }
+
+        public override LineItem ToModel(LineItem lineItem)
+        {
+            base.ToModel(lineItem);
+
+            if (lineItem is DemoOrderLineItem item)
```

Comment:
instead **item** better to call like **demoItem** or more better **demoOrderLineItem**

Reply:
Fixed